instructionAPI/src/InstructionDecoder-amdgpu-vega.C

```diff
@@ -318,7 +309,7 @@ namespace Dyninst {
 
 		void InstructionDecoder_amdgpu_vega::finalizeMTBUFOperands(){
 			layout_mtbuf & layout = insn_layout.mtbuf;
-			const amdgpu_insn_entry  &insn_entry = amdgpu_insn_entry::mtbuf_insn_table[layout.op];
+			//const amdgpu_insn_entry  &insn_entry = amdgpu_insn_entry::mtbuf_insn_table[layout.op];
```

Comment:
If we don't need these variables, let's remove them completely.

Reply:
These variables are removed.